apps/graph/graph/calculation_parameter_controller.cpp

```diff
@@ -108,13 +105,13 @@ int CalculationParameterController::typeAtLocation(int i, int j) {
 void CalculationParameterController::willDisplayCellForIndex(HighlightCell * cell, int index) {
   assert(index >= 0 && index <= numberOfRows());
   if (cell != &m_preimageCell) {
-    I18n::Message titles[] = {I18n::Message::Intersection, I18n::Message::Maximum, I18n::Message::Minimum, I18n::Message::Zeros, I18n::Message::Tangent, I18n::Message::Integral};
+    I18n::Message titles[] = {I18n::Message::Intersection, I18n::Message::Maximum, I18n::Message::Minimum, I18n::Message::Zeros, I18n::Message::Integral};
```

Comment:
Why do you remove the tangent ?
```suggestion
    I18n::Message titles[] = {I18n::Message::Intersection, I18n::Message::Maximum, I18n::Message::Minimum, I18n::Message::Zeros, I18n::Message::Tangent, I18n::Message::Integral};
```

Reply:
Because the tangent is not anymore in the submenu but in the main one

Reply:
OK